bin/cron/deliver_poste_messages

```diff
@@ -127,7 +127,9 @@ class Deliverer
 
     message = StringIO.new
 
-    to_address = parse_address(header['to'], recipient)
+    to_address = parse_address(header['to'], recipient.merge({
+      poste_deliveries_contact_email: recipient[:contact_email]
```

Comment:
Isn't this just copying the value from `:contact_email` to `:poste_deliveries_contact_email`? Why is this necessary? Can't we just reference `defaults[:contact_email']` below?